test/app/components/no_initializer_component.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class NoInitializerComponent < ActionView::Component::Base
+end
```

Comment:
With #175 you should be able to use `MissingInitializerComponent` in the tests and drop guy.